src/blog/2024/09/how-to-build-automate-devops-pipelines-node-red-deployments.md

```diff
@@ -0,0 +1,91 @@
+   - **Prompt to Select Snapshot:** Prompts at deploy time to select which snapshot from the source stage should be copied to the next stage.
+10. Check the option **Deploy to Devices** if you want changes to be deployed to all devices connected to this stage’s instance when the stage is deployed.
+
+Once you’ve created your initial stage, you can add more stages by following the same process. This flexibility allows you to tailor your DevOps pipeline to meet the specific needs of your Node-RED deployment.
+
+![DevOps pipelines animation](./images/devops-pipeline.gif){data-zoomable}
+*Image: DevOps animation showing application deployment to production securely.*
+
+For example, in development, you might have a Node-RED instance in the cloud to build your application. During staging, you could test the setup with a single device. Finally, in production, you can deploy the tested application to thousands of devices in a device group, saving time and ensuring smooth deployment at scale.
+
+## Running a Pipeline Stage
+
+Once your pipeline is set up, you are ready to run it and deploy your changes across each stage. Here's how to do it:
+
+![Screenshot of the 'Run Pipeline' button in FlowFuse, allowing users to trigger the deployment process.](./images/button-to-run-pipeline.png){data-zoomable}
+_Click the 'Run Pipeline' button to initiate the automated deployment process across all configured stages_
+
+1. Click the run pipeline stage option. For each stage, this button will be available, except for the last one. Simply click it to start the deployment process.
+2. Once initiated, the deployment will automatically progress from the leftmost stage to the rightmost stage. It will copy all settings, environments, and flows and deploy them to the next stage. However, for environment variables, only the keys will be copied—you will need to manually add the values for each.
+
+It’s essential to remember that each stage acts as either a source or a target: stages (except the last) push forward, while stages (except the first) receive the results from the previous one. This ensures your deployment flows in an organized, left-to-right manner, with no steps overlooked. Also, it is important to note that when deploying to devices, only one Device Group can be included, and it must be the final stage. This ensures that all changes are fully tested and verified before reaching production, guaranteeing a safe and reliable deployment.
+
+## Conclusion
+
+Setting up DevOps pipelines for Node-RED improves safety and efficiency in industrial deployments. Automation allows teams to quickly test and roll out updates, reducing errors and downtime. This clear process not only makes deployments more reliable but also helps organizations adapt to changes more easily. By using a strong DevOps pipeline, businesses can ensure their critical applications run smoothly, supporting continuous operations in manufacturing and automotive environments.
```

Comment:
```suggestion
DevOps pipelines formalize deployment patterns, standardize testing, and streamline updates, minimizing errors and downtime. This clear process improves reliability and helps organizations adapt more easily thus ensuring smooth operation of critical applications in manufacturing and automotive environments.
```

I _think_ this is a little less wordy/repetitive.  In particular, it removes the "automation" word (deliberately, see prev suggestion)